test/AWS.Logger.SeriLog.Tests/SeriLoggerTestClass.cs

```diff
@@ -50,6 +51,55 @@ public void MultiThreadBufferFullTest()
             MultiThreadBufferFullTestGroup("AWSSeriLogGroupMultiThreadBufferFullTest");
         }
 
+        [Fact]
+        public void RestrictedToMinimumLevelTest()
+        {
+            string logGroupName = "AWSSeriLogGroupRestrictedtoMinimumLevel";
+            // Create logger
+            var configuration = new ConfigurationBuilder()
+            .AddJsonFile("AWSSeriLogGroupRestrictedToMinimumLevel.json")
+            .Build();
+
+            Log.Logger = new LoggerConfiguration().
+                ReadFrom.Configuration(configuration).
+                WriteTo.AWSSeriLog(configuration, null, null, Serilog.Events.LogEventLevel.Error).CreateLogger();

```

Comment:
I'm a little confused by the test design - if the issue this is addressing is for not reading the `restrictedToMinimumLevel` from `appsettings.json` - why are we specifying the minimum level here instead of in the json?

Reply:
@ashovlin Somehow while running through test runner, it doesn't follow the code path it is supposed to follow. While the actual application debugging works fine. You could try getting the branch and see what I mean. I tried it several times but then gave up. We can discuss when I'm back online.

Reply:
I have fixed the issue. The version of `Serilog.Settings.Configuration` needed to be updated (test suite was using the old version having the config issue). Also updated the version of `Serilog` to `2.10.0`.

Reply:
Based on @ppittle's suggestion, min-upgraded Serilog to version 2.6.0 (from 2.5.0) since it was anyway required by old and new (updated). version of Serilog.Settings.Configuration.